### DISCUSSION TITLE
Error: No module named 'deepspeech_pytorch' while creating a PyTorchDeepSpeech() object

### DISCUSSION THREAD

**Discussion by AnupKumarGupta** [Trusted-AI/adversarial-robustness-toolbox#1839] [Q&A] [answered]
**Describe the bug**
While creating a `PyTorchDeepSpeech()`, the following error is shown:

`Error: No module named 'deepspeech_pytorch' `

The error occurs in the file `art/estimators/speech_recognition/pytorch_deep_speech.py` at line 131. 

Please let me know whether I need to install this library or any other workaround.

**To Reproduce**
Steps to reproduce the behaviour:
1. Import `PyTorchDeepSpeech` from `art.estimators.speech_recognition.pytorch_deep_speech`, using the following command:
`from art.estimators.speech_recognition.pytorch_deep_speech import PyTorchDeepSpeech`

2. Call `PyTorchDeepSpeech()`

**Expected behaviour**
Please let me know whether I need to install this library or any other workaround.

**System information (please complete the following information):**
 - OS : `Ubuntu 18.04.6 LTS`
 - Python version: 3.7.13
 - ART version or commit number: [4df35d2](https://github.com/Trusted-AI/adversarial-robustness-toolbox/commit/4df35d28da9af5f0616fdbfdbb16dc94250a745f)
 -  PyTorch: `1.12.1+cu113`

**Comment by beat-buesser** [comment, marked as answer]
Hi @AnupKumarGupta Yes, you need to first install `deep_speech` following the stepson its README page: [deepspeech_pytorch](https://github.com/SeanNaren/deepspeech.pytorch)
